File: Gerontocracy.Web/src/gerontocracy/modules/party/models/politiker-detail.ts
import { ParteiOverview } from './partei-overview';
import { VorfallData } from './vorfall-data';

export interface PolitikerDetail {
    id: number;
    externalId: number;
    name: string;
    wahlkreis: string;
    bundesland: string;
    parteiId?: number;
    partei: ParteiOverview;
    vorfaelle: VorfallData[];
    reputationUp: number;
    reputationDown: number;
    isInactive: boolean;
}

File: Gerontocracy.Core/BusinessObjects/Party/PolitikerDetail.cs
using Gerontocracy.Core.BusinessObjects.Affair;
using System.Collections.Generic;

namespace Gerontocracy.Core.BusinessObjects.Party
{
    public class PolitikerDetail
    {
        public long Id { get; set; }
        public long ExternalId { get; set; }

        public string Name { get; set; }
        public string Wahlkreis { get; set; }
        public string Bundesland { get; set; }
        public bool IsInactive { get; set; }

        public int ReputationUp { get; set; }
        public int ReputationDown { get; set; }

        public long? ParteiId { get; set; }
        public ParteiOverview Partei { get; set; }
        public IEnumerable<VorfallData> Vorfaelle { get; set; }
    }
}
